pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -1077,26 +1089,6 @@ func TestPreFilterStateAddRemovePod(t *testing.T) {
 		expectedAntiAffinity topologyToMatchedTermCount
 		expectedAffinity     topologyToMatchedTermCount
 	}{
-		{
```

Comment:
why was this removed?

Reply:
It's because when no Affinity in incoming Pod, `AddPod` and `RemovePod` shouldn't be called.
They are skipped like Filter plugin. ([ref1](https://github.com/sanposhiho/kubernetes/blob/eabb70833a5649e10ab81f04423ce5cb16aba1b7/pkg/scheduler/framework/runtime/framework.go#L664), [ref2](https://github.com/sanposhiho/kubernetes/blob/eabb70833a5649e10ab81f04423ce5cb16aba1b7/pkg/scheduler/framework/runtime/framework.go#L699))